packages/forms-web-app/src/dynamic-forms/dynamic-components/multi-field-input/question.js

```diff
@@ -117,6 +137,45 @@ class MultiFieldInputQuestion extends Question {
 			}
 		];
 	}
+
+	/**
+	 * returns the formatted answers values to be used to build task list elements
+	 * @param {Journey} journey
+	 * @param {InputField[]} inputFields
+	 * @returns {String}
+	 */
+	formatContactDetails(journey, inputFields) {
+		const firstNameField =
```

Comment:
could this logic be held in input fields somehow so that it is generic
something like, or is that too complex?

foreach inputfield
format += inputField.formatForSummary(answers)

Reply:
See above - I've amended to use a formatJoinString field on InputField, with a '\n' default.